modules/doc/content/application_development/build_system.md

```diff
@@ -44,80 +49,88 @@ some of the environment variables you can set and their effects:
   an advanced option and there are very few reasons to disable this. You will likely notice a
   dramatic slowdown if you set this to false.
 
-## Build Methods (Creating "debug" Builds)
+## Build Methods
 
+!style! halign=left
 As mentioned above there are two environment variables that control what "type" of executable gets
 created: `METHOD` and `METHODS`.  The executable that is ultimately created will have the `$METHOD`
-as a suffix like: `something-$METHOD`.  The valid options are:
+as a suffix like: `something-$METHOD`. The valid options are:
+!style-end!
 
 - `opt`: (The default) Builds an optimized executable suitable for running calculations.
-- `dbg`: An executable meant for use in a debugger (like `gdb` or `lldb`).  Will be very slow and not
-  meant to be used on large problems
+- `dbg`: An executable meant for use in a debugger (like `gdb` or `lldb`).  Will be very slow and
+  not meant to be used on large problems
 - `devel`: Something in-between `opt` and `dbg`.  Attempts to be faster but also does some more
   run-time checks (asserts and mooseAsserts)
 - `oprof`: Not normally used.  Can be used for "code profiling": running your code with a utility
   like `oprof`, `gprof`, `vtune`, etc.
 
-`METHODS` is used to specify the full set (space separated) of the above options that you will _ever_
-want to used.  It is used in the build of `libMesh` to build a `libMesh` library for each of the
-`METHODS`.
+`METHODS` is used to specify the full set (space separated) of the above options that you will
+ _ever_ want to used.  It is used in the build of `libMesh` to build a `libMesh` library for each of
+ the `METHODS`.
 
-## Build System Optimization
+## Build Optimization
 
+!style! halign=left
```

Comment:
```suggestion
!alert! note
```

Let's make this stand out a bit more since the section isn't strictly necessary.